src/Ombi/ClientApp/src/app/my-nav/my-nav.component.scss

```
@@ -1,260 +1,326 @@
-    flex:0 0 250px;
-}
+    gap: 6px;
+    height: 30px;
+    padding: 0 14px;
+    border-radius: 15px;
+    border: 1px solid rgba(255, 255, 255, 0.12);
+    background: transparent;
+    color: rgba(255, 255, 255, 0.55);
+    font-size: 12px;
+    font-weight: 500;
+    font-family: 'Inter', 'Roboto', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)